.github/workflows/request-reviews.yml

```diff
@@ -0,0 +1,24 @@
+name: Request reviews for PRs
+
+on:
+  pull_request_target:
+    types: [opened]
+
+permissions:
+  contents: read
+  pull-requests: write
+  issues: write
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29  # v4.1.6
+        name: Checkout default branch
+      - uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea  # v7.0.1
+        name: Make initial comment
+        with:
+          script: |
```

Comment:
You can check if the PR author is `github` bot here.

Reply:
This workflow also replaces part of the GitHub bot functionality, and checking for that at this level wouldn't work with everything else